docs/.eslintrc.js

```diff
@@ -25,5 +25,6 @@ module.exports = {
     "react/require-default-props": "off",
     "react/no-array-index-key": "off",
     "import/no-extraneous-dependencies": "off",
+    "react/no-danger": "off",
```

Comment:
I'd consider `dangerouslySetInnerHTML` as a dangerous practice and unless we have a strong reason for that I wouldn't allow to use it everywhere. As I see we have only one place when we need this. Can we disable eslilnt for that case precisely?